infra/modules/service/events_role.tf

```diff
@@ -30,27 +30,35 @@ resource "aws_iam_policy" "run_task" {
 
 data "aws_iam_policy_document" "run_task" {
```

Comment:
This file looks *exactly* like the one for the scheduler role:

https://github.com/navapbc/platform-test/blob/main/infra/modules/service/scheduler_role.tf

But it didn't seem right to share them, given that they are acting on different arns, and are assumed by different services.